examples/imageviewer/mapview-desktop/src/desktopMain/kotlin/example/map/Config.kt

```diff
@@ -0,0 +1,27 @@
+package example.map
+
+val TILE_SIZE = 256
+
+object Config {
+    val OPENSTREET_MAP_LICENSE: String = "https://wiki.openstreetmap.org/wiki/OpenStreetMap_License"
+    val OPENSTREET_MAP_POLICY: String = "https://operations.osmfoundation.org/policies/"
+    val CLICK_DURATION_MS: Long = 300
+    val CLICK_AREA_RADIUS_PX: Int = 7
+    val ZOOM_ON_CLICK = 0.8
+    val MAX_SCALE_ON_SINGLE_ZOOM_EVENT = 2.0
+    val CACHE_DIR_NAME = "map-view-cache"
+    val SCROLL_SENSITIVITY_DESKTOP = 0.05
+    val MIN_ZOOM = 0
+    val MAX_ZOOM = 22
+    val FONT_LEVEL = 2
```

Comment:
Please add comments what these constants are used for

Reply:
Done